pandas/core/frame.py

```diff
@@ -2659,8 +2659,11 @@ def assign(self, **kwargs):
         \*\*kwargs. For python 3.5 and earlier, since \*\*kwargs is unordered,
         the columns are inserted in alphabetical order at the end of your
         DataFrame.  Assigning multiple columns within the same ``assign``
-        is possible, but you cannot reference other columns created within
-        the same ``assign`` call.
+        is possible, but for python 3.5 and eralier you cannot reference other
+        columns created within the same ``assign`` call. For python 3.6 and
+        above it is possible to reference columns created in an assignment.
+        To this end you have to respect the order of |*|*kwargs and use
+        callables referencing the assigned columns.
```

Comment:
python --> Python
eralier --> earlier, <-- note the comma

Add comma after "To this end"

Reply:
sorry I fear the |*|*kwargs part is an artifact of somewhere else, will also change the earlier typo